packs/azure/actions/lib/formatters.py

```diff
@@ -0,0 +1,16 @@
+__all__ = [
+    'to_node_dict'
+]
+
+
+def to_node_dict(node):
```

Comment:
We need to come up with a good pattern / solution for handling common code which is needed in multiple packs which doesn't result in us copy and pasting code all over the place.

Currently, the same code is needed in libcloud, softlayer and azure pack.

I was thinking about creating a new Python package - `st2contrib-libcloud-formatters` or something along those lines, but I'm not too happy with this approach.

Feedback and suggestions are welcome.